tests/functional/test_sanity.py

```diff
@@ -10,9 +7,7 @@ def test_sanity(self, app):
     def test_we_can_read_and_send_params(self, app):
         # We aren't really running this test because we care, but just
         # to prove the testing framework
-        result = app.get(
-            "/flags/test", params=self.json_fixture("sanity/bar_on.json"), status=200
-        )
+        result = app.get("/flags/test", params={"feature_flags.bar": True}, status=200)
```

Comment:
`params={"feature_flags.bar": True}` is a lot clearer than `params=self.json_fixture("sanity/bar_on.json")`, which required you to understand what `self.json_fixture()` was (you wouldn't find it in the file: it was defined in a base class in another file that the test class inherited from), find the `tests/functional/fixtures/sanity/bar_on.json` file, and open it to find `{"feature_flags.bar": true}`.